Vates/VatesSimpleGui/ViewWidgets/src/MdViewerWidget.cpp

```diff
@@ -871,14 +860,12 @@ MdViewerWidget::checkViewAgainstWorkspace(ModeControlWidget::Views view,
   if (VatesViewerInterface::MDHW == workspaceType) {
     // Histo workspaces cannot have a splatter plot,
     if (view == ModeControlWidget::SPLATTERPLOT) {
-      g_log.notice()
-          << "The preferred initial view favours the splatterplot as initial "
-             "view, "
-          << "but an MDHisto workspace is being loaded. An MDHisto workspace "
-          << "cannot be loaded into a splatterplot view. Defaulted to standard "
-             "view. \n";
-
-      selectedView = ModeControlWidget::STANDARD;
+      g_log.notice("The preferred initial view favours the splatterplot "
+                   "as initial view, but an MDHisto workspace is being "
+                   "loaded. A MDHisto workspace cannot be loaded into a "
+                   "splatterplot view. Defaulted to MultiSlice view.");
+
+      selectedView = ModeControlWidget::MULTISLICE;
```

Comment:
This change will affect all MDHisto workspaces where the user tries to load them initially into Splatterplot mode -- not only Single Crystal Diffraction. Is this intended?